projects/cashmere/src/lib/modal/modal-options.ts

```diff
@@ -1,7 +1,7 @@
 export type ModalSize = 'sm' | 'md' | 'lg' | 'xl';
 export interface ModalOptions {
     /** size of the modal window. Defaults to medium */
```

Comment:
Hey @joeskeen - to keep the documentation accurate, can you adjust that comment above to say that this field is optional as well.